lib/index.js

```diff
@@ -510,9 +561,21 @@ var getNav = function(navName, config, navFiles, settings, metadata){
 
     nodes = addNodeMetadata(nodes, navFiles, config, metadata);
 
-    if(mergeMatchingFilesAndDirs){
-        nodes = mergeFileDirs(nodes);
+    if(cleanUrls){
+        if(mergeIndexWithParentDir || mergeMatchingFilesAndDirs){
```

Comment:
why is `mergeMatchingFilesAndDirs` here?

Reply:
It seems like the `cleanUrls` setting is not needed. The readme should explain that you use `mergeIndexWithParentDir` and not `mergeMatchingFilesAndDirs ` for the clean urls plugin.